apps/condo/domains/common/components/UnitInfo.tsx

```diff
@@ -0,0 +1,64 @@
+import { useIntl } from '@core/next/intl'
```

Comment:
could you please move this component to the nearest domain? Maybe it's the property. The common domain is for non-domain logic and components